package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -0,0 +1,215 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+Diffusion map --- :mod:`MDAnalysis.analysis.diffusionmap`
+=====================================================================
+
+:Author: Eugen Hruska
+:Year: 2016
+:Copyright: GNU Public License v3
+
+The module contains the non-linear dimension reduction method diffusion map.
+The diffusion map allows to get a quick estimate of the slowest collective 
+coordinates for a trajectory. This non-linear dimension reduction method 
+assumes that the trajectory is long enough to represents a probability 
+distribution of as protein close to the equilibrium. Further the diffusion map 
+assumes that the diffusion coefficients are constant. The eigenvectors with the 
+largest eigenvalues are the slowest collective coordinates. The complexity of 
+the diffusion map is O(N^2), where N is the number of frames in the trajectory. 
+Instead of a single trajectory a sample of protein structures can be used. 
+The sample should be equiblibrated, at least locally.  Different weights can be used. 
+The order of the sampled structures in the trajectory is irrelevant.
+
+More details how the correst slowest collective coordinates can be calculated.in:
+[1] Nadler, B, Lafon, S, Coifman, R. R, & Kevrekidis, I. G. (2013) Diffusion maps, 
```

Comment:
Make the references a separate section at the end of the background named References and use [reST citations](http://www.sphinx-doc.org/en/stable/rest.html#citations).